tests/test_flair_inference.py

```diff
@@ -0,0 +1,144 @@
+        train_domains=["reviews"],
+        dev_domains=["reviews"],
+        test_domains=["reviews"],
+        text_cfg="sentence",
+    )
+    data_loader = HuggingFaceDataLoader()
+    transformation = ClassificationCorpusTransformation("text", "target").then(
+        DownsampleFlairCorpusTransformation(*(0.01, 0.01, 0.01), stratify=False)
+    )
+    embedding = AutoFlairDocumentEmbedding.from_hub("allegro/herbert-base-cased")
+    task = TextClassification(output_path.name, task_train_kwargs={"max_epochs": 1})
+    model = FlairModel(embedding, task)
+    evaluator = TextClassificationEvaluator()
+    pipeline = StandardPipeline(dataset, data_loader, transformation, model, evaluator)
+    return pipeline
+
+
+@pytest.fixture(scope="module")
+def sequence_labeling_pipeline(
+    tmp_path_module: "TemporaryDirectory[str]",
+) -> StandardPipeline[
+    str, datasets.DatasetDict, Corpus, Dict[str, nptyping.NDArray[Any]], Dict[str, Any]
+]:
+    output_path = tmp_path_module
+    dataset = Dataset("clarin-pl/kpwr-ner")
+    data_loader = HuggingFaceDataLoader()
+    transformation = (
+        ColumnCorpusTransformation("tokens", "ner")
+        .then(SampleSplitsFlairCorpusTransformation(dev_fraction=0.1, seed=441))
+        .then(DownsampleFlairCorpusTransformation(*(0.005, 0.01, 0.01), stratify=False))
+    )
+    task = SequenceLabeling(
+        output_path.name,
+        hidden_size=256,
+        task_train_kwargs={"max_epochs": 1, "mini_batch_size": 64},
+    )
+    embedding_loader = FlairWordEmbeddingLoader(
+        "allegro/herbert-base-cased", "model_type_reference"
+    )
+    embedding = embedding_loader.get_embedding()
+    model = FlairModel(embedding, task)
+    evaluator = SequenceLabelingEvaluator()
+    pipeline = StandardPipeline(dataset, data_loader, transformation, model, evaluator)
+    return pipeline
+
+
+def test_text_classification_inference(
+    text_classification_pipeline: StandardPipeline[
+        str, datasets.DatasetDict, Corpus, Dict[str, nptyping.NDArray[Any]], Dict[str, Any]
+    ],
+    tmp_path_module: "TemporaryDirectory[str]",
+) -> None:
+    flair.set_seed(441)
+    output_path = tmp_path_module
+    pipeline = text_classification_pipeline
+    result = pipeline.run()
+
+    task_from_ckpt = TextClassification.from_checkpoint(
+        checkpoint_path=(Path(output_path.name) / "final-model.pt"), output_path=output_path.name
+    )
+    loaded_data = pipeline.data_loader.load(pipeline.dataset)
+    transformed_data = pipeline.transformation.transform(loaded_data)
+    test_data = transformed_data.test
+
+    y_pred, loss = task_from_ckpt.predict(test_data)
+    y_true = task_from_ckpt.get_y(test_data, task_from_ckpt.y_type, task_from_ckpt.y_dictionary)
+    results_from_ckpt = pipeline.evaluator.evaluate({"y_pred": y_pred, "y_true": y_true})
+    assert np.array_equal(result["data"]["y_pred"], results_from_ckpt["data"]["y_pred"])
+
+
+def test_sequence_labeling_inference(
```

Comment:
can we append these tests to `tests/test_flair..` where the model is already trained? It will speed up tests.

Reply:
ok